modelo/DDL.sql

```diff
@@ -0,0 +1,44 @@
+;
+CREATE DATABASE  sql_app;
+\c sql_app;
+
+CREATE TABLE users(
```

Comment:
Dúvida:

Pelo oq eu entendi esse SQL é rodado pelo seu sistema pra inicializar o DB, certo?

Oq acontece se seu sistema cair, subir de novo, e o DB já tiver a tabela lá?
Dá algum erro?

Just in case, algo do tipo resolveria esse potencial BO:
```suggestion
CREATE TABLE IF NOT EXISTS users(
```

Referência:
https://www.postgresql.org/docs/current/sql-createtable.html

Reply:
Na verdade, para essa atividade em si, eu rodei esses comandos manualmente. Entendi o risco que voce apontou. Vou aplicar sim.